packages/dds/merge-tree/src/ordinal.ts

```diff
@@ -19,13 +19,18 @@ export function computeHierarchicalOrdinal(
 
 	const ordinalWidth = 1 << (maxCount - actualCount);
 	let ordinal: string;
-	if (previousOrdinal === undefined || previousOrdinal === "") {
-		ordinal = parentOrdinal + String.fromCodePoint(ordinalWidth - 1);
+	if (previousOrdinal === undefined) {
+		// Since ordinals don't deal with user input and would require minor behavior changes,
```

Comment:
I know I suggested basically this, but I'd probably put something a bit more technically precise in the code :-)

Perhaps this as the justification?
```suggestion
		// Ordinals exist purely for lexicographical sort order and use a small set of valid bytes for each string character.
		// The extra handling fromCodePoint has for things like surrogate pairs is therefore unnecessary.
```